libvncclient/listen.c

```diff
@@ -30,11 +30,12 @@
 #endif
 #include <sys/types.h>
 #ifdef WIN32
-#define close closesocket
```

Comment:
Point taken. It leaves me wondering though why that error shows up. After all, ```closesocket()``` should be defined at all at that point. I noticed that in [libvncserver/sockets.c](https://github.com/LibVNC/libvncserver/blob/2041bfd5cd6c6a8bcec260a3fa86d340050892c8/libvncserver/sockets.c#L116) ```#define closesocket close``` is used while in [libvncclient/sockets.c](https://github.com/LibVNC/libvncserver/blob/2041bfd5cd6c6a8bcec260a3fa86d340050892c8/libvncclient/sockets.c#L44)  it's the other way around. Maybe symbols leak from the server when compiling the client... Anyway, your patch passes Travis on Linux, I can't prove it's not working, so I won't complain anymore ;)